graphql/schemabuilder/pagination.go

```diff
@@ -316,6 +316,36 @@ func getEdges(key string, nodes []interface{}) (edges []Edge) {
 	return edges
 }
 
+// Creates a pages slice, starting with a blank cursor, then every n+1 edge's cursor (if you have 20
+// entries per page, 19, 39, 59 etc). This works for `after:` but works unexpectedly for `before:`
+
+// NOTE: The cursors are based off of the total and are not relative to the current query, meaning
+// that they will shift with each query as entries are added.
+func getPages(edges []Edge, limit int) (pages []string) {
```

Comment:
`pagesFromEdges`? should this one be on the `connectionContext` as well?

Reply:
👍